crates/sui-storage/src/lib.rs

```diff
@@ -15,7 +16,6 @@ use std::fs::File;
 use std::io::{BufReader, Read, Write};
 use std::path::PathBuf;
 use std::{fs, io};
-use sui_simulator::fastcrypto::hash::{HashFunction, Sha3_256};
```

Comment:
@sadhansood / @mystenmark, is this a reasonable change?

I couldn't tell whether it was intentional or not, but it seemed weird for us to include `sui-simulator` in our regular builds.

Reply:
This seems reasonable to me. I think fastcrypto could be directly used here